secure_redis/client.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
```

Comment:
why do we need to include a client at all here? why not just indicate that we should be specifying the SERIALIZER setting, plus the REDIS_SECRET_KEY setting